src/ray/raylet/node_manager.cc

```diff
@@ -1253,6 +1256,37 @@ void NodeManager::ProcessWaitRequestMessage(
   RAY_CHECK_OK(status);
 }
 
+void NodeManager::ProcessWaitForDirectActorCallArgsRequestMessage(
```

Comment:
Can you rename this to something more general like `AsyncWait`? Not really tied to DAC at all.